spec/factories/participant_profiles/ects.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :ect, class: "ParticipantProfile::ECT" do
```

Comment:
there is already a `ect_participant_profile` factory. is that insufficient for your needs?